.env.example

```diff
@@ -2,3 +2,7 @@ NODE_ENV=
 PORT=
 GOOGLE_CLIENT_ID=
 GOOGLE_CLIENT_SECRET=
+
+POSTGRES_PASSWORD=
```

Comment:
We are going with MySQL as agreed during our call last night though we will be using Sequelize ORM to interface with the DB. This will be updated in the next PR @ekundayo-ab.